TileDB.CSharp/TileDB.CSharp.csproj

```diff
@@ -14,39 +18,39 @@
   </ItemGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64' and '$(OS)' == 'Windows_NT'">
-    <OutputPath>../dist/Windows/lib</OutputPath>
+    <OutputPath>./lib</OutputPath>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64' and '$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Linux)))'">
-    <OutputPath>../dist/Linux/lib</OutputPath>
+    <OutputPath>./lib</OutputPath>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64' and '$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::OSX)))'">
-    <OutputPath>../dist/Darwin/lib</OutputPath>
+    <OutputPath>./lib</OutputPath>
   </PropertyGroup>
 
   <ItemGroup Condition=" '$(OS)' == 'Windows_NT' ">
-    <Content Include="..\dist\Windows\lib\tiledb.dll" CopyToOutputDirectory="PreserveNewest">
+    <Content Include=".\lib\tiledb.dll" CopyToOutputDirectory="PreserveNewest">
       <Link>tiledb.dll</Link>
     </Content>
-    <Content Include="..\dist\Windows\lib\tiledbcs.dll" CopyToOutputDirectory="PreserveNewest">
+    <Content Include=".\lib\tiledbcs.dll" CopyToOutputDirectory="PreserveNewest">
       <Link>tiledbcs.dll</Link>
     </Content>
   </ItemGroup>
   <ItemGroup Condition=" '$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::OSX)))' ">
-    <Content Include="..\dist\Darwin\lib\libtiledb.dylib" CopyToOutputDirectory="PreserveNewest">
+    <Content Include=".\lib\libtiledb.dylib" CopyToOutputDirectory="PreserveNewest">
       <Link>libtiledb.dylib</Link>
     </Content>
-    <Content Include="..\dist\Darwin\lib\tiledbcs.dylib" CopyToOutputDirectory="PreserveNewest">
-      <Link>libtiledbcs.dylib</Link>
+    <Content Include=".\lib\tiledbcs.dylib" CopyToOutputDirectory="PreserveNewest">
+      <Link>tiledbcs.dylib</Link>
     </Content>
   </ItemGroup>
-  <ItemGroup Condition=" '$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Linux)))' And Exists('..\dist\Linux\lib\libtiledb.so')">
+  <ItemGroup Condition=" '$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Linux)))'">
      <!-- this item is conditional in case we have a system-wide install -->
-     <Content Include="..\dist\Linux\lib\libtiledb.so" CopyToOutputDirectory="PreserveNewest">
-      <Link>tiledb.so</Link>
+     <Content Include=".\lib\libtiledb.so.2.4" CopyToOutputDirectory="PreserveNewest">
```

Comment:
Do we need the unversioned so here? or are we directly linking `libtiledb.so.2.4`?